docs/user-guide.md

```diff
@@ -131,19 +133,19 @@ the values that are assigned to the variable. For example:
 @NullMarked
 public class MyClass {
   void myMethod(@Nullable String one, String two) {
-    String copyOfOne = one;
-    String copyOfTwo = two;
+    String aliasOfOne = one;
```

Comment:
Nothing is copied in these examples, so `alias` seems like a better term.

Reply:
or maybe just `anotherOne`